src/_posts/2019-06-12-progress-structure.md

```diff
@@ -0,0 +1,52 @@
+
+At Codurance there was some basic structure: Junior, Craftsperson, Principal, but that structure was barely defined. We kind of knew how a person moved from Junior to Craftsperson, but we did not know how you could move from Craftsperson—and this is a problem. To understand more, I recommend that you watch this [presentation](https://youtu.be/jA1Q94d2z10?list=PLBzScQzZ83I_qiY6iuS-jHmp1QvdE8m5_) by [Marco Rodgers](https://www.linkedin.com/in/marcorogers/) or look at his [slides](https://speakerdeck.com/polotek/creating-a-career-ladder-for-engineers) or do both. But the gist of it is that, without a clear idea of how to move, people get lost in the middle. It was an issue for our juniors because they were left to their own devices to know how to improve, and for senior people because they were left without a way to show their advancement, nor a way to reach a higher position.
+
+So out of these concerns we tried earlier to create some kind of structure, but the effort fizzled out. Yet the concern was important enough that we tried a second time, giving ourselves tight deadlines to make sure that we set it up. So a group of people got together and we created a progression structure.
+
+One of the things that we decided at the beginning is that we were not trying to create the perfect structure. We wanted to create something that was good enough, and we will keep evaluating the system and improving it once we started to see the issues with it. 
+
+The other important aspect is that we are a consultancy, and therefore the skills chosen for advancement are to improve our capacity as consultants.
+
+## Progress Structure.
+So how is the progress structure setup? We have a collection of disciplines—areas where we want people to improve. Each discipline is divided into at most 5 levels. Each level has one or more skills/experiences. The idea is that they represent a continuous advancement on the difficulty or knowledge required. Then there are the spider nets, an approximate shape of the levels on each discipline that we expect a person at each level to have reached. We want to allow for some flexibility as some of the “requirements” really depend on being provided with opportunities to develop in these areas. Sometimes people are so well advanced around a specific area that compensates for lack in others. But in general, for a consultancy, well rounded individuals, that can cover all the job/projects possibilities is better.
+
+A detail that I believe is important is that your career advancement will not depend on someone of higher seniority level vacating a position. Each person progress is its own, and will not be affected by others progression.
```

Comment:
"and will not be affected by others progression." -> missing posessive apostrophe on "others", but I'd recommend restructuring to something like "and will not be affected by the progression of their peers".

Reply:
Added colleagues, rather than peers.